docs/get-started/installation/prepare/orange-pi-3-lts.md

```diff
@@ -0,0 +1,88 @@
+- [SD card reader](https://www.amazon.com/Reader-Beikell-Connector-Memory-Adapter/dp/B0BGNZGDTC/) (recommended with USB-C connector)
+- A monitor: for display
+- [HDMI cable](https://www.amazon.com/Highwings-Braided-Cord-Supports-ARC-Compatible-Ethernet/dp/B07TDH11BJ/): to connect Orange Pi to monitor display
+- USB keyboard and mouse
+- [USB-C to USB-A cables](https://www.amazon.com/Anker-2-Pack-Premium-Samsung-Galaxy/dp/B07DD5YHMH/): to connect keyboard and mouse to USB hub or Orange Pi ports (recommended, hardware dependent)
+- [USB hub](https://www.amazon.com/BYEASY-Extended-Portable-Splitter-MacBook/dp/B07TVH9NHP/) with USB-A connector (recommended)
+
+## Power your Orange Pi 3 LTS
+
+Before you power the board, you need to install an operating system.
+
+1. First, download an [Orange Pi 3 LTS Ubuntu image](https://drive.google.com/drive/folders/1KzyzyByev-fpZat7yvgYz1omOqFFqt1k) to your development machine.
+   We recommend `ubuntu_jammy_desktop`.
+1. Unzip the image.
+1. Insert the micro-SD card into the SD card reader and connect the reader to your development machine.
+1. Follow [this guide](https://sbc-community.org/docs/general_guides/prepare_sd_card/) to flash the OS to your micro-SD card using [balenaEtcher](https://etcher.balena.io/).
+1. Insert the micro-SD into the Orange Pi.
+
+To power your Orange Pi 3 LTS, connect your power adapter to the LTS's USB-C port.
+The LED should light up, which indicates that the board is powered.
+
+{{% alert title="Tip" color="tip" %}}
+If your board's LED does not light up when powered, try re-flashing your micro-SD card with the `ubuntu_jammy_desktop` image or using a different micro-SD card.
+The Orange Pi will only power on with a compatible operating system installed.
+{{% /alert %}}
+
+To connect to a display, connect the HDMI cable to the Orange Pi's HDMI port and the other end to your monitor.
+Then, to connect the keyboard and mouse, plug the keyboard and mouse into the two USB-A ports on your Orange Pi.
+Alternatively, plug the two devices into your USB hub and connect the USB hub to your Orange Pi's USB-A port.
+
+{{% alert title="Tip" color="tip" %}}
+For board schematics, consult the [Orange Pi 3 LTS documentation](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/orange-pi-3-LTS.html).
+{{% /alert %}}
+
+After it boots, you should be greeted with the Orange Pi desktop display.
+If it prompts you for a password, note that the default password for Orange Pi devices is "orangepi".
+
+## Establish a network connection
+
+The Orange Pi 3 LTS comes equipped with a wireless network antenna.
+To connect to WiFi through the desktop, click on the WiFi icon in the top right of the monitor display, select your preferred network or hotspot, and enter the password.
+
+{{% alert title="Tip" color="tip" %}}
+You can also connect to WiFi while connected to the Orange Pi on the terminal with several different methods, including the `nmcli` tool.
```

Comment:
Same opt nit as below: phrasing:

```suggestion
You can also connect to Wifi using command-line tools, such as `nmcli`, from the terminal on your Orange Pi.
This is useful if you are booting your Orange Pi without a GUI, or are configuring it remotely using `ssh` over a wired network connection.
```

Reply:
Here I did mean to also account for SSH connections. But the several different methods phrasing I would keep bc on the server image terminal display (which I mistakenly entered) you can also enter an interface from the terminal? It's rather odd

Reply:
Ah gotcha, like a TUI (text-user-interface) thing? Funnily enough, `nmcli` also has one called [`nmtui`](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-configuring_ip_networking_with_nmtui). (It sucks. I do not miss the days I spent wrestling with `nmcli` and RedHat's system boot logic (surely we don't need network up before we look up DNS names for filesystem mounts over the network, right?))

Right on, ignore my comment then, thanks for clarifying. 👍